### DISCUSSION TITLE
Scraping dynamic pages

### DISCUSSION THREAD

**Discussion by julzKB** [FreshRSS/FreshRSS#4888] [Q&A] [answered]
Hi there,

I'm having issues scraping this Metacritic page :

[Metacritic ](https://www.metacritic.com/browse/games/score/metascore/90day/switch/filtered
)

I assume that's because it's a dynamic page and it must rendered somehow, is it possible to do in FreshRSS alone ? Or maybe I could connect to a browser in a container and steal the renderer source from there . Not sure how people do this in FreshRSS but I would be interested in hearing your ideas .

Thank you

**Comment by Alkarex** [comment, marked as answer]
Hello,
It is not a dynamic page (you can verify that by using cURL or looking at the source code in your browser - not the DOM inspector), but a page that is restricting access based on parameters such as user-agent.

Try to change the user-agent for that feed in FreshRSS and use the [one from your own Web browser](https://www.whatismybrowser.com/detect/what-is-my-user-agent/).

![image](https://user-images.githubusercontent.com/1008324/203496558-b86a4d06-36e2-4380-91f4-3de697936347.png)

And avoid refreshing too often.

**Comment by julzKB** [comment]
yep ! that was it, user error :) Thank you
